### PR TITLE
Git version info

### PR DESCRIPTION
**Pull request by chiragsanghvi**
This **PR** allows to generate a **version.txt** file on `make install` using [git_version_info.sh](https://github.com/informatics-isi-edu/chaise/blob/git_version_info/git_version_info.sh). It contains the git log for **chaise** and **ermrestjs** version which is being used. 

You can browse this file in the browser using the **Chaise** App url *https://[CHAISE_URL]/chaise/version.txt*. 

The file **version.txt** looks like this
```
**************************CHAISE BUILD INFO******************************

BUILD DATE : Fri Aug  5 10:35:08 PDT 2016

CHAISE : git_version_info*ncommit 367de825223ec301c906713e09407684351a8fa7
Author: Chirag Sanghvi <csanghvi@appacitive.com>
Date:   Fri Aug 5 10:29:43 2016 -0700

    added bower install in travis.yml

EMRESTJS : masterncommit 7fc961bf8be9d040fdedddeeb3b0aad543c497b4
Author: Chirag Sanghvi <csanghvi@appacitive.com>
Date:   Thu Aug 4 15:33:18 2016 -0700

    Fixed failing http retry testcase

```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -52,6 +52,7 @@ before_install:
   - git clone https://github.com/informatics-isi-edu/ermrestjs.git
   - cd ermrestjs
   - sudo npm install
+  - bower install
```
Comment (karlcz) [inline .travis.yml]: What does this have to do with the rest of this PR?  Was it committed by accident?

Comment (chiragsanghvi) [inline .travis.yml]: @karlcz  Nope, I added this line to install **ermrestjs**  **bower** dependencies when deploying CHAISE on TRAVIS. 

This is a part of Ermrestjs **PR** [#136](https://github.com/informatics-isi-edu/ermrestjs/pull/136) and has no implications on existing CHAISE code.

**Comment by robes** [comment]
This seems quite wrong to me.

1. one project (chaise) is generating version numbers for other projects (ermrest, ermrestjs)
2. it assumes that the other repose will be on the same server as this repo and in a particular relative location to this repo
3. unnecessary verbosity of this
4. script could be simpler

**Comment by chiragsanghvi** [comment]
@robes I understand what you mean. We can do it that way for other projects too instead of including all info about **ermrest** and **ermrestjs** in **chaise**. 

I just wanted to have a one stop place for viewing the current git log, so made this script part of **chaise**.

**Comment by hongsudt** [comment]
I asked Chirag for a simple page (or a page footer) that contains commit
hashes about our deployment (full stack, if possible), so we can check the
software version without logging into the actual server.

Can we discuss a proper way to obtain this information on Monday? Maybe
ermrest and other server component (e.g. web authn, hatrac) need to expose
their version numbers through a REST api.
